packages/iris-grid/src/IrisGridTreeTableModel.js

```diff
@@ -32,6 +37,35 @@ class IrisGridTreeTableModel extends IrisGridTableModel {
     };
   }
 
+  snapshot(ranges) {
```

Comment:
This still needs to be async, it needs to match the signature of the super IrisGridTableModel
Side note IrisGridModel should update the signature to match IrisGridTableModel as well